packages/app/src/vis-packs/core/hooks.ts

```diff
@@ -160,3 +161,34 @@ export function useSlicedDimsAndMapping(
     [dimMapping, dims]
   );
 }
+
+function useFillValue(dataset: Dataset): number | undefined {
+  const { attrValuesStore } = useDataContext();
+  const rawFillValue = attrValuesStore.getSingle(dataset, '_FillValue');
+
+  if (typeof rawFillValue === 'number') {
+    return rawFillValue;
+  }
+
+  if (!Array.isArray(rawFillValue)) {
+    return undefined;
+  }
+  const fillValue = rawFillValue[0];
+
+  return typeof fillValue === 'number' ? fillValue : undefined;
+}
+
+export function useIgnoreValue(
```

Comment:
> this is like `getModifierKeyArray` so it might be worth abstracting this into a generic `wrap` utillity of sorts :shrug: 

Or use [castArray](https://lodash.com/docs/4.17.15#castArray) ?

>  const fillValue = typeof wrappedFillValue[0] === 'number' ? new Float32Array(wrappedFillValue)[0] : undefined;

Not sure if you meant that `Float32Array` should be used everywhere but I think it is needed to derive the `TypedArray` from the dataset `type`

Reply:
ah yeah, perfect!